src/main/java/org/broadinstitute/dsm/util/externalShipper/GBFRequestUtil.java

```diff
@@ -389,6 +390,22 @@ public void orderCancellation(ArrayList<KitRequest> kitRequests) throws Exceptio
         return kitRequests;
     }
 
+    public static  List<String> getDSMKitRequestId(@NonNull Connection conn, String externalOrderNumber) {
+        List<String> dsmKitRequestIds = new ArrayList<>();
+        try (PreparedStatement stmt = conn.prepareStatement(SQL_SELECT_KIT_REQUEST_BY_EXTERNAL_ORDER_NUMBER)) {
+            stmt.setString(1, externalOrderNumber);
+            try (ResultSet rs = stmt.executeQuery()) {
+                while (rs.next()) {
+                    dsmKitRequestIds.add(rs.getString(DBConstants.DSM_KIT_REQUEST_ID));
+                }
+            }
+        }
+        catch (SQLException ex) {
+            throw new RuntimeException("Error looking up kit requests  ", ex);
```

Comment:
Maybe add `externalOrderNumber` to this exception text so we know a bit more context when debugging?